src/Altinn.App.Core/Infrastructure/Clients/Profile/ProfileClient.cs

```diff
@@ -61,12 +68,38 @@ public class ProfileClient : IProfileClient
         _appMetadata = appMetadata;
         _accessTokenGenerator = accessTokenGenerator;
         _telemetry = telemetry;
+
+        _cache = new MemoryCache(new MemoryCacheOptions());
+        _cacheEntryOptions = new MemoryCacheEntryOptions()
+        {
+            AbsoluteExpirationRelativeToNow = TimeSpan.FromSeconds(cacheSettings.Value.ProfileCacheLifetimeSeconds)
+        };
     }
 
```

Comment:
The old code gets the memory cache from DI.

Won't this approach disable the cache in an app when the service is registered with `.AddHttpClient<>`?